src/models/game_session.ts

```diff
@@ -240,7 +246,7 @@ export default class GameSession {
 
         // manage recently played song queue
         if (guildPreference.getShuffleType() === ShuffleType.UNIQUE && Math.min(guildPreference.getLimit(), totalSongs) === this.lastPlayedSongsQueue.length) {
-            logger.info(`${getDebugContext(message)} | Resetting lastPlayedSongsQueue (all ${guildPreference.getLimit()} unique songs played)`);
+            logger.info(`gid: ${this.guildID} | Resetting lastPlayedSongsQueue (all ${guildPreference.getLimit()} unique songs played)`);
```

Comment:
You can probably overload `getDebugContext` to take in a `MessageContext` and print out this

Reply:
I don't think I can export the same function name twice. An alternative is to make `getDebugContext`'s parameter of type `any`, and then use `instanceof` to decide what to return.

Reply:
Probably union type